packages/domain/src/general/offers.ts

```diff
@@ -134,7 +134,7 @@ export const LocationStateToArray = z
   .transform((previous) => {
     const deprecatedLocationStateFormat = LocationState.safeParse(previous)
     if (deprecatedLocationStateFormat.success) {
-      return [deprecatedLocationStateFormat]
+      return [deprecatedLocationStateFormat.data]
```

Comment:
This is good catch, but eventually we need to introduce system that does not lets us do mistakes like these. It should be typechecked. But here it's tricky